sdcm/tester.py

```diff
@@ -3390,6 +3392,7 @@ def check_regression_with_baseline(self, subtest_baseline):
                                                                     is_gce=is_gce,
                                                                     extra_jobs_to_compare=self.params.get('perf_extra_jobs_to_compare'))
         except Exception as exc:  # noqa: BLE001
+            self.log.exception("Error while checking regression")
             TestFrameworkEvent(
                 message='Failed to check regression',
                 source=self.__class__.__name__,
```

Comment:
you can add tracback to the event, I think it would be more useful, isn't it ? 
```suggestion
            TestFrameworkEvent(
                message='Failed to check regression',
                trace=sys._getframe().f_back,
                source=self.__class__.__name__,
```

Reply:
Yes, that is better.
Amended in https://github.com/scylladb/scylla-cluster-tests/commit/bbf0bcd8e8666824fe54f46eb31172c304fcd267